.github/workflows/publish.yml

```diff
@@ -17,4 +17,24 @@ jobs:
           DOCKER_USER: ${{secrets.DOCKER_USER}}
           DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
       - name : push to Docker Hub
-        run: make build-image image-push
\ No newline at end of file
+        run: make build-image image-push
+  push-artifact-hub:
+    runs-on: ubuntu-20.04
```

Comment:
We should use ubuntu-20.04 instead of ubuntu-latest ?

Reply:
Did you mean the contrary ?

Reply:
Ok